### PR TITLE
sample for Point-to-Site connection for Virtual Network Gateway

### PR DESCRIPTION
**Pull request by lenala**

### CODE REVIEW

File: azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+                    .withAddressSpace("192.168.0.0/16")
+                    .withAddressSpace("10.254.0.0/16")
+                    .withSubnet("GatewaySubnet", "192.168.200.0/24")
+                    .withSubnet("FrontEnd", "192.168.1.0/24")
+                    .withSubnet("BackEnd", "10.254.1.0/24")
+                    .create();
+            System.out.println("Created network");
+            // Print the virtual network
+            Utils.print(network);
+
```
Comment (anuchandy) [inline azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java]: For consistency with other samples can we add the comment

//============================================================
// Create virtual network gateway

File: azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+            System.out.println("Creating virtual network gateway...");
+            VirtualNetworkGateway vngw1 = azure.virtualNetworkGateways().define(vpnGatewayName)
+                    .withRegion(region)
+                    .withExistingResourceGroup(rgName)
+                    .withExistingNetwork(network)
+                    .withRouteBasedVpn()
+                    .withSku(VirtualNetworkGatewaySkuName.VPN_GW1)
+                    .create();
+            System.out.println("Created virtual network gateway");
+
```
Comment (anuchandy) [inline azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java]: same comment as above

File: azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+ *  - Generate and download VPN client configuration package. Now it can be used to create VPN connection to Azure.
+ *  - Revoke a client certificate
+ *
+ *  Please note: in order to run this sample, you need to have:
+ *   - pre-generated root certificate and public key exported to $CERT_PATH file
+ *      For more details please see https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-certificates-point-to-site for PowerShell instructions
+ *      and https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-certificates-point-to-site-makecert for Makecert instructions.
+ *   - client certificate generated for this root certificate installed on your machine.
+ *      Please see: https://docs.microsoft.com/en-us/azure/vpn-gateway/point-to-site-how-to-vpn-client-install-azure-cert
+ *   - thumbprint for client certificate saved to $CLIENT_CERT_THUMBPRINT
```
Comment (anuchandy) [inline azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java]: to be consistent with wording for `$CERT_PATH`.. "thumbprint for client certificate exported to $CLIENT_CERT_THUMBPRINT"

File: azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java (hunk 1 above)
Comment (lenala) [reply azure-samples/src/main/java/com/microsoft/azure/management/network/samples/ManageVpnGatewayPoint2SiteConnection.java]: Fixed.

**Pull request merged**